src/components/Disputes/DisputeDetail.js

```diff
@@ -64,50 +65,65 @@ const DisputeDetail = React.memo(function DisputeDetail({ match }) {
       <Bar>
         <BackButton onClick={handleBack} />
       </Bar>
-      <Split
-        primary={
-          <React.Fragment>
-            <DisputeInfo
-              id={disputeId}
-              dispute={dispute}
-              loading={disputeFetching}
-              onDraft={actions.draft}
-              onRequestCommit={requests.commit}
-              onRequestReveal={requests.reveal}
-              onLeak={actions.leak}
-              onRequestAppeal={requests.appeal}
-              onExecuteRuling={actions.executeRuling}
-            />
-            {(() => {
-              if (disputeFetching) {
-                return null
-              }
-              if (evidences.length === 0) {
-                return <NoEvidence />
-              }
-              return (
-                // TODO- in next PR will get plaintiff and deffendant from the dispute
-                <DisputeEvidences
-                  evidences={evidences}
-                  plaintiff={creatorAddress}
-                  defendant=""
-                />
-              )
-            })()}
-          </React.Fragment>
-        }
-        secondary={
-          <React.Fragment>
-            <Box heading="Dispute timeline" padding={0}>
-              {disputeFetching ? (
-                <div css="height: 200px" />
-              ) : (
-                <DisputeTimeline dispute={dispute} />
-              )}
-            </Box>
-          </React.Fragment>
+
+      {(() => {
```

Comment:
Nitpick, but we could lift `DisputeInfoComponent` out of the render so we don't need the IIFE here:

```js
</Bar>
{dispute?.status === DisputeStatus.Voided ? DisputeInfoComponent : (
  ...
)}
```